sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-shared.service.ts

```diff
@@ -444,11 +454,11 @@ export class DisbursementScheduleSharedService extends RecordDataModelService<Di
     applicationNumber: string,
     entityManager: EntityManager,
   ): Promise<Record<string, number>> {
-    // Get already sent (disbursed) values to know the amount that the student already received.
+    // Get already ready to send or sent (disbursed) values to know the amount that the student already received.
```

Comment:
do we need `already` here? `Get ready to send and sent (disbursed) values to know the amount that the student already received.` ?

Reply:
Changed as suggested.